### PR TITLE
Fix PubNub issue

### PR DESCRIPTION
**Pull request by tylerlong**
Fix https://github.com/ringcentral/ringcentral-php/issues/69

### CODE REVIEW

File: src/Subscription/Subscription.php [hunk 1]
```diff
@@ -445,27 +445,11 @@ public function pubnubTimeoutHandler()
      */
     public function notify($pubnubMessage)
     {
-
-        if (!empty($pubnubMessage['error'])) {
```
Comment (kirill-konshin) [inline src/Subscription/Subscription.php]: Is it safe to drop this functionality?

**Comment by tylerlong** [comment]
I also need to fix the tests. Because we mocked `$pubnubMessage` as array but it is not.

I need to spend more time to figure out why it was array? Maybe it's the behavior of PubNub 3.x?

**Comment by kirill-konshin** [comment]
Very weird. Yes please adjust the tests, make sure we're not loosing any functionality. It could be due to Pubnub 3, and the errors in demo were suppressed by PHP error config, technically it allows to access object properties as if it is an array, it can throw a notice/warning in this case, it all depends on error reporting settings.

**Comment by byrnereese** [comment]
@tylerlong @kirill-konshin I am encountering this problem as well. It is currently exhibiting itself in our PHP documentation. Is there a status or alternative syntax one can use?

**Comment by tylerlong** [comment]
Hi all, if you have no concern, I am going to change the mock of `$pubnubMessage`. It is wrongly mocked as an array while it should be an object instead.

But if you remember there are some special reasons for it to be an array(in order to make things work), please do let me know.

@kirill-konshin

**Comment by kirill-konshin** [comment]
No objections

**Comment by tylerlong** [comment]
Hi all, unit tests fixed.

**Comment by tylerlong** [comment]
I find that unit test for HHVM still fails.

But it fails when doing `make install` which has nothing to do with this PR.

@kirill-konshin  Could you please take a look?

**Comment by kirill-konshin** [comment]
Just remove it from the list. I think we never had a statement that we support it officially.

**Pull request merged**